capstone/src/main/java/com/google/sps/data/User.java

```diff
@@ -1,12 +1,43 @@
 package com.google.sps.data;
-import java.io.*;
-import java.util.List;
-
-public interface User {
-    public void userInfo(String email, String nickname, String id);
-    public void setNickname(String newNickname);
-    public String getEmail();
-    public List<Classroom> getClassrooms();
-    public String getNickname();
-    public int getId();
-}
+
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.service.DatabaseService;
+
+public class User {
+    protected static final String NICKNAME = "nickname";
+    protected static final String USER = "User";
```

Comment:
This looks like the only Uppercase string-key, is that on purpose?

Reply:
Yes, this is kind of entity so I made it uppercase

Reply:
Having a naming convention on the constants makes that more clear. So something like NICKNAME_PROPERTY_KEY and USER_ENTITY_NAME or whatever else would help readability